File: 1980-find-unique-binary-string/1980-find-unique-binary-string.py
class Solution:
    def findDifferentBinaryString(self, nums: List[str]) -> str:
        hset = set(nums)
        nums.sort()
        n = len(nums)
        start = '0' * n
        while(start in nums):
            val = int(start, 2)
            start = bin(val + 1)[2:]
            start = '0' * (n - len(start)) + start
        
        return start
            

File: 1980-find-unique-binary-string/1980-find-unique-binary-string.go
func findDifferentBinaryString(nums []string) string {
    ans := []rune{}
    for i := 0; i < len(nums); i++ {
        curr := nums[i][i]
        add := '1' 
        if curr != '0' {
            add = '0'
        }
        ans = append(ans, add)
    }
    return string(ans)
}